polyglot/piranha/test-resources/go/feature_flag/builtin_rules/boolean_expression_simplify/configurations/piranha_arguments.toml

```diff
@@ -0,0 +1,17 @@
+# Copyright (c) 2022 Uber Technologies, Inc.
+#
+# <p>Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file
+# except in compliance with the License. You may obtain a copy of the License at
+# <p>http://www.apache.org/licenses/LICENSE-2.0
+#
+# <p>Unless required by applicable law or agreed to in writing, software distributed under the
+# License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing permissions and
+# limitations under the License.
+
+language = ["go"]
+substitutions = [
+    ["true_flag_name", "true"],
```

Comment:
can we standardize this to `treated` and `treated_complement` ? Look at Java/Kt implementation

Reply:
sounds good

Reply:
please rename to `treated/treated_complement/nil`